addons/web/static/src/model/relational_model/datapoint.js

```diff
@@ -40,6 +41,8 @@ export class DataPoint extends Reactive {
         this.id = getId("datapoint");
         this.model = model;
         this._config = config;
+        this._config.activeFields = markRaw(config.activeFields);
+        this._config.fields = markRaw(config.fields);
```

Comment:
```suggestion
        markRaw(config.activeFields);
        markRaw(config.fields);
        this._config = config;
```
markRaw acts in place. Should probably contain a brief comment explaining why these can and should be raw